src/Editor Guidelines.sln

```diff
@@ -37,24 +37,45 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "ColumnGuide", "ColumnGuide\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VSIX", "VSIX\VSIX.csproj", "{5ADE1B3F-BC61-42F3-A467-F782D4EFF07E}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VSIX_Dev17", "VSIX_Dev17\VSIX_Dev17.csproj", "{FEB7F80E-E565-4119-A18E-D2B9F5E95383}"
+EndProject
 Global
 	GlobalSection(SharedMSBuildProjectFiles) = preSolution
 		ColumnGuide\ColumnGuide.projitems*{5ade1b3f-bc61-42f3-a467-f782d4eff07e}*SharedItemsImports = 5
 		ColumnGuide\ColumnGuide.projitems*{61d4b2d7-4433-4c71-be24-57a36758ed99}*SharedItemsImports = 13
+		ColumnGuide\ColumnGuide.projitems*{feb7f80e-e565-4119-a18e-d2b9f5e95383}*SharedItemsImports = 5
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x86 = Debug|x86
```

Comment:
💡 Probably don't want the additional solution platform

Reply:
Good idea. See #78